drivers/SmartThings/sonos/src/sonos_driver.lua

```diff
@@ -644,6 +664,7 @@ function SonosDriver.new_driver_template()
     oauth_token_bus = oauth_token_bus,
     oauth_info_bus = oauth_info_bus,
     oauth = {},
+    have_alerted_unauthorized = false,
```

Comment:
Is the intention that this notification can happen at most once per driver restart?